### PR TITLE
Tree Picker: Item sorting

### PR DESCRIPTION
**Pull request by leekelleher**
Adds sorting feature to the Document/Media/Member Picker selected items.

### CODE REVIEW

File: src/packages/core/repository/repository-items.manager.ts [hunk 1]
```diff
@@ -76,10 +87,25 @@ export class UmbRepositoryItemsManager<ItemType> extends UmbBaseController {
...
+					this.#items.next(this.#sortByUniques(data));
+				},
+				'_observeRequestedItems',
+			);
 		}
 	}
 
+	#sortByUniques(data: Array<ItemType>) {
+		const uniques = this.getUniques();
+		return [...data].sort((a, b) => {
```
Comment (bjarnef) [inline src/packages/core/repository/repository-items.manager.ts:102]: Can't we use this here?

```
var uniqueAndSorted = [...new Set(myData)].sort() 
```

https://stackoverflow.com/a/42495707

Comment (leekelleher) [reply src/packages/core/repository/repository-items.manager.ts:102]: Thanks @bjarnef.  The intent of the `#sortByUniques` method is to sort the `data` array (of objects) in the order of the unique values. More like this: https://stackoverflow.com/q/13304543/12787

Comment (bjarnef) [reply src/packages/core/repository/repository-items.manager.ts:102]: Ahh okay, I haven't checked if `Set()` eventually can sort unique data objects in array:
https://stackoverflow.com/a/43667135

**Review by nielslyngsoe — APPROVED** [review]
Cool

**Pull request merged**